src/tasks/CustomMenusMigrationTask.php

```diff
@@ -31,11 +32,13 @@ class CustomMenusMigrationTask extends MigrationTask
      */
     public function up()
     {
-        
+        $config = SiteConfig::current_site_config();
         $menus = CustomMenuHolder::get();
         $i = 0;
         
         foreach ($menus as $menu) {
+            $config->Menus()->add($menu);
```

Comment:
We can't just blindly add menu's to current siteconfig. We need to check if the menu is assigned first (for example, on subsites, menus can be assigned to multiple siteconfigs).

The menu should only be assigned if not already assigned to a siteconfig (possible `!$menu->SiteConfig()->exists()`, or whatever is the correct syntax)?